skorch/classifier.py

```diff
@@ -266,12 +266,28 @@ def _default_callbacks(self):
             ('print_log', PrintLog()),
         ]
 
+    @property
+    def classes_(self):
+        return [0, 1]
+
     # pylint: disable=signature-differs
     def check_data(self, X, y):
         super().check_data(X, y)
         if get_dim(y) != 1:
             raise ValueError("The target data should be 1-dimensional.")
 
+    def infer(self, x, **fit_params):
+        y_infer = super().infer(x, **fit_params)
+        if y_infer.dim() < 2:
```

Comment:
If we want to be really direct:

```suggestion
        if y_infer.dim() == 1:
```